libs/dh/market-participant/actors/feature-actors/src/lib/create/dh-actors-create-actor-modal.component.ts

```diff
@@ -238,4 +248,85 @@ export class DhActorsCreateActorModalComponent {
   close() {
     this.innerModal?.close(false);
   }
+
+  createMarketParticipent(): void {
+    this._apollo
+      .mutate({
+        mutation: CreateMarketParticipantDocument,
+        variables: {
+          input: {
+            userInvite: {
+              email: this.newUserForm.controls.email.value,
+              firstName: this.newUserForm.controls.firstName.value,
+              lastName: this.newUserForm.controls.lastName.value,
+              phoneNumber: this.newUserForm.controls.phone.value,
+              assignedRoles: this._userSelectedUserRoles.map((userRole) => userRole.id),
+              assignedActor: 'f3df856f-bd11-4174-97cb-fb6bc54c300a',
```

Comment:
issue: hard-coded value.

Reply:
They are needed but, cannot be provided. API needs to be changed at a later date